images/linux/scripts/SoftwareReport/SoftwareReport.Common.psm1

```diff
@@ -3,6 +3,11 @@ function Get-BashVersion {
     return "Bash $version"
 }
 
+function Get-DashVersion {
```

Comment:
I'd mention the main purpose of dash in the image as well (`/bin/sh` replacement) just for clarity.